VirtualReality/qSlicerHomeVirtualWidget.cxx

```diff
@@ -1,93 +0,0 @@
-/*==============================================================================
```

Comment:
Did you actually delete the widget???

Reply:
Not sure how this happened, I don't remember deleting these files but they indeed aren't in my /VirtualReality/ directory. All I can think of is that we mistakenly deleted these when when you came by on Monday. Anyways I've taken them from your SlicerVR fork and put them back